main.go

```diff
@@ -24,12 +26,17 @@ func main() {
 	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: ghTkn})
 	cl := oauth2.NewClient(oauth2.NoContext, ts)
 	ghClient := github.NewClient(cl)
+	dockerCl, err := dlib.NewClientFromEnv()
```

Comment:
Will this work with Docker For Mac?  (probably not as it's looking for the docker-machine env vars, huh?)

We may consider explicitly mentioning the need for docker-machine now that multiple Mac options are available...